app.js

```diff
@@ -42,10 +59,106 @@ const shopRoutes = require("./routes/shop");
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "public")));
 
+// sequelizes
+/*
+app.use((req, res, next) => {
+  User.findByPk(1)
+    .then((user) => {
+      req.user = user;
+      next();
+    })
+    .catch((err) => console.log("getUserError:", err));
+});
+*/
+
+/*
+app.use((req, res, next) => {
+  User.findById("628ae7a5419d06f23392185e")
+    .then((user) => {
+      console.log(user);
+      req.user = new User(user.name, user.email, user.cart, user._id);
+      next();
+    })
+    .catch((err) => console.log(err));
+});
+*/
+
+app.use((req, res, next) => {
+  User.findById("628b043a66380d08755312c3")
+    .then((user) => {
+      req.user = user;
+      next();
+    })
+    .catch((err) => console.log(err));
+});
+
 app.use("/admin", adminRoutes.router);
 app.use(shopRoutes);
 
 app.use(errorController.get404);
 
-// Create server in port 3000 using express as a handler
-app.listen(3000);
+// sequelizes
+/*
+Product.belongsTo(User, { constraints: true, onDelete: "CASCADE" });
+User.hasMany(Product);
+User.hasOne(Cart);
+Cart.belongsTo(User);
+Cart.belongsToMany(Product, { through: CartItem });
+Product.belongsToMany(Cart, { through: CartItem });
+Order.belongsTo(User);
+User.hasMany(Order);
+Order.belongsToMany(Product, { through: OrderItem });
+
+sequelize
+  // .sync({ force: true })
+  .sync({})
+  .then((result) => {
+    return User.findByPk(1);
+  })
+  .then((user) => {
+    if (!user) {
+      return User.create({ name: "Galang", email: "test@test.com" });
+    }
+    return user;
+  })
+  .then((user) => {
+    return user.createCart();
+  })
+  .then((cart) => {
+    // Create server in port 3000 using express as a handler
+    app.listen(3000);
+  })
+  .catch((err) => {
+    console.log("sequelizeError:", err);
+  });
+*/
+
+// MongoDB
+/*
+mongoConnect(() => {
+  app.listen(3000);
+});
+*/
+
+mongoose
+  .connect(
+    "mongodb+srv://galang:galang-cluster-0@clusternode0.gqomc.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
try to use this package - https://www.npmjs.com/package/dotenv

and put credentials in separate file